front/app/containers/Admin/index.tsx

```diff
@@ -132,12 +132,14 @@ const AdminPage = memo<Props & WithRouterProps>(
     const noPadding =
       adminNoPadding ||
       pathname.includes('admin/dashboard') ||
+      pathname.includes('admin/initiatives') ||
       pathname.includes('admin/reporting');
 
     const fullWidth =
       adminFullWidth === true ||
       endsWith(pathname, 'admin/moderation') ||
       pathname.includes('admin/dashboard') ||
+      pathname.includes('admin/initiatives') ||
```

Comment:
Is it still used? On my machine, it doesn't take up the full width 
![image](https://user-images.githubusercontent.com/91883434/234238572-3a00a8a5-7e7f-404a-96a9-062557073dd8.png)

Reply:
Indeed, it looks different on a FullHD laptop 
![image](https://user-images.githubusercontent.com/91883434/234245852-704a371f-e7d5-4ed5-b748-7aed9d5e0971.png)

The previous screenshot is taken on a 4k monitor with 1.5x scale.

Reply:
This is what it looks like on my laptop screen. I'm going to leave a note in our test ticket to assess / review appearance on different screen sizes.
<img width="1508" alt="image" src="https://user-images.githubusercontent.com/12659000/234247933-4e4f3ff3-2947-4f21-910c-4e9d96665afe.png">